<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>historyForm</class>
 <widget class="QWidget" name="historyForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>106</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="topVerticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="historyCheckBox">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Zapisuj historię tłumaczeń</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="middleVerticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSpinBox" name="noOfItemsSpinBox">
         <property name="frame">
          <bool>true</bool>
         </property>
         <property name="prefix">
          <string/>
         </property>
         <property name="minimum">
          <number>5</number>
         </property>
         <property name="maximum">
          <number>25</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="noOfItemsLabel">
         <property name="toolTip">
          <string/>
         </property>
         <property name="text">
          <string>Maksymalna liczba pozycji</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="bottomVerticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>historyCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>noOfItemsSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>43</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>historyCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>noOfItemsLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
